### ISSUE TITLE
Change Login System

### ISSUE THREAD

**Issue by websnooze** [splewis/get5-web#125]
Hello,

### Expected behavior
Is it possible to change the login system ? Add a custom login page without Steam Auth to restrict users ?

### Actual behavior
The login system is with steam and all players can login to the Get5 WebInterface...

- Plugin version: get5_0.6.0
- Sourcemod version: 1.9.0.6265

**Comment by tloesch** [comment]
If you want a custom login then you need to build it yourself.

But you can already restrict Steam Users. Take a look in the configuration file.
There is a whitelist. Just add allowed SteamIds to it.

**Comment by 66venum** [comment]
Oh Okay I found it, is it on prod_config.py ?

Like this WHITELISTED_IDS = [76561198184313278, 76561198184313279] ?

**Comment by tloesch** [comment]
Yes in prod_config.py

I dont have an example right now. But it should be:

`WHITELISTED_IDS = ['76561198184313278', '76561198184313279']`

**Comment by TandelK** [comment]
Wait just understand the usage between Admins and Whitelisted IDs . If you are hosting kind of Tournament and have different Server Managers and Admins , you can have 2 different sections 

With ADMINS_ACCESS_ALL_MATCHES = True 
It will become way better functions for Admins to allow to access other Matches created by Managers in case of emergencies. Also very useful if you dont want Managers to create their own new teams and servers everytime. 


Basically Admins can add Public Servers and Teams which can be accessed by Managers and keep a track on it. While if the Manager creates a Server and Match it can only be access by only him and wont allow 2nd manager in case the first manager gets disconnected or have any other issues.
